app/src/main/java/com/veriff/demo/data/LoginResponse.kt

```diff
@@ -0,0 +1,17 @@
+package com.veriff.demo.data
+
+import com.google.gson.annotations.SerializedName
+
+data class LoginResponse(
+        @SerializedName("token_type")
+        val tokenType: String = "",
```

Comment:
wouldn't this field still be `null` when Gson deserializes an empty object like `{}`? I think the type should probably be `String?` not `String`.

Reply:
For empty object it will set to the default value "", but its a good point to make it optional incase it comes as null.